lib/api/fragments/content-blocks.js

```diff
@@ -31,15 +31,12 @@ export const calloutBlockFragment = `
           backgroundColor
           dynamicComponent
           header
-          image {
-            ...on calloutImages_Asset {
-              ${getImageFields("fit", 750)}
-            }            
-            ...on cantoDam_Asset {
-              id
-              url
-              width
-              height
+          damAsset {
```

Comment:
removed the duplicate interface querying, preserving backwards compatibility here is creating type issues that prevent successful builds.